programs/svm-spoke/src/instructions/refund_claims.rs

```diff
@@ -26,11 +26,7 @@ pub struct InitializeClaimAccount<'info> {
     pub system_program: Program<'info, System>,
 }
 
-pub fn initialize_claim_account(
-    ctx: Context<InitializeClaimAccount>,
-    mint: Pubkey,
-    token_account: Pubkey
-) -> Result<()> {
+pub fn initialize_claim_account(ctx: Context<InitializeClaimAccount>, _: Pubkey, _: Pubkey) -> Result<()> {
```

Comment:
we can delete the unused props here and rename them to `_mint` and  `_token_account` in the outer function.